### DISCUSSION TITLE
Works like a dream

### DISCUSSION THREAD

**Discussion by DFKSingleton** [nlohmann/json#2657] [General]
I needed a JSON serialiser for C++ and came across this project from my Internet research. Downloaded the development zip. It took me a little bit of time to realise that I only needed the amalgamated header. After including the header in my code, it worked like a dream! Very simple to use and the examples are good. I should add that my requirements were very simple, just a simple data structure with a few entries.

**Comment by nlohmann** [comment]
Thanks a lot! If you want to support me and my work on this project, you can become a sponsor on GitHub (https://github.com/sponsors/nlohmann), or use Paypal (http://paypal.me/nlohmann).
